custom_components/luxtronik/sensor.py

```diff
@@ -915,6 +916,22 @@ def native_value(self):  # -> float | int | None:
                 return LUX_STATUS1_HEATPUMP_SHUTDOWN
             # endregion Workaround Luxtronik Bug: Line 1 shows 'heatpump coming' on shutdown!
 
+        
+        # workaround to detect (passive) cooling active
+        if (self._sensor_key == LUX_SENSOR_STATUS) and (value == LUX_STATUS_NO_REQUEST):     
+            if self._luxtronik.detect_cooling_present():
+                T_in       = self._luxtronik.get_value("calculations.ID_WEB_Temperatur_TVL")
```

Comment:
The var names break the HA Style guidelines.
https://developers.home-assistant.io/docs/development_guidelines/
Pleas use
temp_out
and so on.